tests/system/test_cluster/test_integrity_sync/test_integrity_sync.py

```diff
@@ -171,12 +171,14 @@ def test_extra_files(clean_files):
             assert result == '', f"File {file} was expected to be removed from {host}, but it still exists."
 
 
-def test_extra_valid_files(clean_files):
+def not_test_extra_valid_files(clean_files):
     """Check that extra_valid files created in the workers are copied in the master.
 
     Register two agents, wait and check that the client.keys file have been updated in all worker nodes.
     Then, each worker creates an 'agent-groups' file for one of the created agents. Files inside
     /var/ossec/queue/agent-groups are 'extra_valid' according to cluster.json, so they should be sent to the master.
+
+    This test is no longer in use, but it could be useful in the future.
```

Comment:
We should delete this test if it is no longer in use. The test already exists in `4.3` if we need it in the future.